src/components/SpaceDetailCard/SpaceDetailCard.vue

```diff
@@ -38,10 +38,17 @@
         class="space-detail-card__facilities"
       />
 
-      <div class="flat-list space-detail-card__seating">
-        <SvgIcon name="seat-icon" class="space-detail-card__seating-icon" />
-        {{ space.seats }} {{ $t("seatsDescription") }}
-      </div>
+      <ul class="flat-list space-detail-card__seating">
+        <li>
+          <SvgIcon name="seat-icon" class="space-detail-card__seating-icon" />
+          {{ space.seats }} {{ $t("seatsDescription") }}
+        </li>
+        <ClientOnly>
```

Comment:
Can we make the `<ClientOnly>` part of the `<OccupancyIndicator>`? To me that's an internal of the component and avoids the need to wrap it everywhere it's used. In Nuxt2 I think you could do this by prepending `Lazy` to a component. Not sure if Nuxt3 has a similar mechanism.

Reply:
Currently `<OccupancyIndicator>` is there mostly as a hint that we might have some common piece of UI to display occupancy in the future. At the moment we just display slightly different and very simple markup in every place where we need occupancy data. So the `<OccupancyIndicator>` component isn't even used everywhere. If the UI would stay like this, I'd consider removing it totally, but we'll definitely have something more complex and nice later on, so let it stay as a placeholder for that I guess. The template of the `<OccupancyIndicator>` itself is already wrapped in `<ClientOnly>`.
A lazy component is a slightly different thing as far as I understand, it's a shortcut to dynamically import the component, extracting its code to a separate JS bundle and postponing its setup. The naming convention for that still works in Nuxt 3.